### PR TITLE
[ch22441]Fix tsc errors

### PR DESCRIPTION
**Pull request by lajoskovago**
[ch22441]Fix tsc errors

### CODE REVIEW

File: common/models/intervention.types.ts [hunk 1]
```diff
@@ -125,6 +125,7 @@ export class InterventionSupplyItem {
   total_price: number | undefined = undefined;
   other_mentions = '';
   intervention = '';
+  outputs: string[] = [];
```
Comment (acory) [inline common/models/intervention.types.ts:128]: 'outputs' doesn't exist on the intervention model, please remove from here and from the other file where the tsc was

File: intervention-management/final-review/final-review-popup.ts [hunk 2]
```diff
@@ -70,7 +70,7 @@ export class FinalReviewPopup extends LitElement {
 
   processRequest(): void {
     // validate if file is selected for new attachments
-    if (!this.data && !this.data.attachment) {
+    if (!this.data && !this.data!.attachment) {
```
Comment (acory) [inline intervention-management/final-review/final-review-popup.ts:73]: it seems like it should be  if (!this.data || !this.data.attachment)

Comment (lajoskovago) [reply intervention-management/final-review/final-review-popup.ts:73]: it does not affect lint warnings, this should be another card to review the whole process

Comment (acory) [reply intervention-management/final-review/final-review-popup.ts:73]: as I suggested in the meeting

**Review by acory — APPROVED** [review]

**Pull request merged**